examples/wolf_sheep_experimental/model.py

```diff
@@ -0,0 +1,132 @@
+"""
+Wolf-Sheep Predation Model
+================================
+
+Replication of the model found in NetLogo:
+    Wilensky, U. (1997). NetLogo Wolf Sheep Predation model.
+    http://ccl.northwestern.edu/netlogo/models/WolfSheepPredation.
+    Center for Connected Learning and Computer-Based Modeling,
+    Northwestern University, Evanston, IL.
+"""
+
+import sys
+import mesa
+from mesa import Model
+from mesa.time import RandomActivation
+from mesa.space import MultiGrid
+from mesa.datacollection import DataCollector
+
+sys.path.append("mesa-examples\\examples\\epstein_civil_violence_experimental")
+
+from agents import GrassPatch, Sheep, Wolf
```

Comment:
I think this should be `.agents`
```suggestion
from .agents import GrassPatch, Sheep, Wolf
```

Reply:
but if directly import the wolf_sheep_experimental in our device and use from 
```
.agents import GrassPatch, Sheep, Wolf
```
its not working and shows import error that's why i used this approach

```
sys.path.append("mesa-examples\\examples\\epstein_civil_violence_experimental")

from agents import GrassPatch, Sheep, Wolf
```

Reply:
@rht do you know how this exactly works?

Reply:
If there is agents.py in the current directory where the script is run from, `from agents import ...` should work without modifying `sys.path`. But `agents` is such a generic name that may overshadow e.g. https://agentpy.readthedocs.io/en/latest/overview.html if it were to be named `agents`.